### PR TITLE
Fixing CI travis build when any test fails.

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@flopezag @jframos 

### DESCRIPTION
- Fixing CI travis build when any test fails.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -27,12 +27,12 @@
 # to be shown in sonar
 #
 # __author__ = 'fla'
-
+set -e
 
 mkdir /var/log/fiware-facts
```
Comment (flopezag) [inline build.sh]: you do not need to execute this sentence with sudo?

File: build.sh [hunk 2]
```diff
@@ -65,6 +65,9 @@ fi
...
 nosetests -s -v --cover-package=facts --with-cover --cover-xml-file=target/site/cobertura/coverage.xml --cover-xml --with-xunit --xunit-file=target/surefire-reports/TEST-nosetests.xml
-sudo /sbin/service rabbitmq-server stop
-kill -9 $(lsof -t -i:5000)
-kill -9 $(lsof -t -i:6379)
+
+if [[ ! $1 == "travis_build" ]];
+then
+    sudo /sbin/service rabbitmq-server stop
+    kill -9 $(lsof -t -i:6379)
+fi
```
Comment (flopezag) [inline build.sh]: new line

**Comment by jframos** [comment]
Coveralls is working! :dancer: :dancer:

File: build.sh (hunk 1 above)
Comment (geonexus) [inline build.sh]: No, it's no needed

**Comment by jframos** [comment]
LGTM

**Comment by flopezag** [comment]
LGTM

**Pull request merged**